src/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -30,6 +30,9 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
+  <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.6'">
+    <AdditionalFiles Include="$(ToolsDir)PinvokeAnalyzer_OneCoreApis.txt" />
+  </ItemGroup>
```

Comment:
Without it (at least when I added it) it prevents the build errors regarding calling native bcrypt APIs. The tools didn't automatically add that for 1.6